pulsar-client-api/src/main/java/org/apache/pulsar/client/api/Consumer.java

```diff
@@ -77,6 +77,8 @@
      *
      * <p>This calls blocks until a message is available.
      *
+     * <p>When thread is Interrupted, return a null value and reset interrupted flag.
```

Comment:
@zjyingchang It seems we throw an exception instead of return `null` on interrupted?

Reply:
Well. I can see we return null in `internalReceive(long timeout, TimeUnit unit)` and `internalBatchReceive()` but not `internalReceive()`. Weird..